src/services/llms/gemini.py

```diff
@@ -0,0 +1,50 @@
+from typing import List, Dict
+import google.generativeai as Client
+from unidiff import Hunk, PatchedFile
+from ...core.config import Config
+from ...core.models import PRDetails
+from .base import BaseLLMService
+
+class GeminiService(BaseLLMService):
+    """
+    Implementation of BaseLLMService for Google's Gemini model.
+    """
+    
+    def __init__(self):
+        """Initialize the Gemini service with configuration."""
+        Client.configure(api_key=Config.GEMINI_API_KEY)
+        self.model = Client.GenerativeModel(Config.GEMINI_MODEL)
+        
+    def create_prompt(self, file: PatchedFile, hunk: Hunk, pr_details: PRDetails) -> str:
+        """Create a prompt formatted for Gemini's expectations."""
+        return f"""Your task is reviewing pull requests. Instructions:
+        - Provide the response in following JSON format: {{"reviews": [{{"lineNumber": <line_number>, "reviewComment": "<review comment>"}}]}}
+        - Provide comments and suggestions ONLY if there is something to improve, otherwise "reviews" should be an empty array.
+        - Use GitHub Markdown in comments
+        - Focus on bugs, security issues, and performance problems
+        - IMPORTANT: NEVER suggest adding comments to the code
+
+        Review the following code diff in the file "{file.path}" and take the pull request title and description into account.
+        Pull request title: {pr_details.title}
+        Pull request description:
```

Comment:
The `get_ai_response` function catches a generic `Exception`. This is too broad and can mask important errors. It's better to catch more specific exceptions (like `google.generativeai.exception.GenerativeAIError`) to handle errors more gracefully and provide more informative error messages.  Consider logging the full exception traceback for debugging purposes, rather than just printing a message to the console.  The current `print` statement might not be visible in a production environment.